docs/docker.md

```diff
@@ -17,325 +17,96 @@ Once you have Docker installed, simply prepare the config file (e.g. `config.jso
+    - The following section assumes that `docker` and `docker-compose` are installed and available to the logged in user.
+    - All below comands use relative directories and will have to be executed from the directory containing the `docker-compose.yml` file.
+    
 
 ### Docker quick start
 
 Create a new directory and place the [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) in this directory.
 
-``` bash
-mkdir ft_userdata
-cd ft_userdata/
-# Download the docker-compose file from the repository
-curl https://raw.githubusercontent.com/freqtrade/freqtrade/develop/docker-compose.yml -o docker-compose.yml
+=== "PC/MAC/Linux"
+    ``` bash
+    mkdir ft_userdata
+    cd ft_userdata/
+    # Download the docker-compose file from the repository
+    curl https://raw.githubusercontent.com/freqtrade/freqtrade/master/docker-compose.yml -o docker-compose.yml
 
-# Pull the freqtrade image
-docker-compose pull
+    # Pull the freqtrade image
+    docker-compose pull
 
-# Create user directory structure
-docker-compose run --rm freqtrade create-userdir --userdir user_data
+    # Create user directory structure
+    docker-compose run --rm freqtrade create-userdir --userdir user_data
 
-# Create configuration - Requires answering interactive questions
-docker-compose run --rm freqtrade new-config --config user_data/config.json
-```
+    # Create configuration - Requires answering interactive questions
+    docker-compose run --rm freqtrade new-config --config user_data/config.json
+    ```
 
-The above snippet creates a new directory called "ft_userdata", downloads the latest compose file and pulls the freqtrade image.
-The last 2 steps in the snippet create the directory with user-data, as well as (interactively) the default configuration based on your selections.
+=== "RaspberryPi"
+    ``` bash
+    mkdir ft_userdata
+    cd ft_userdata/
+    # Download the docker-compose file from the repository
+    curl https://raw.githubusercontent.com/freqtrade/freqtrade/master_pi/docker-compose.yml -o docker-compose.yml
 
-!!! Note
+    # Pull the freqtrade image
+    docker-compose pull
+
+    # Create user directory structure
+    docker-compose run --rm freqtrade create-userdir --userdir user_data
+
+    # Create configuration - Requires answering interactive questions
+    docker-compose run --rm freqtrade new-config --config user_data/config.json
+    ```
+
+The above snippet creates a new directory called `ft_userdata`, downloads the latest compose file and pulls the freqtrade image.
+The last 2 steps in the snippet create the directory with `user_data`, as well as (interactively) the default configuration based on your selections.
+
+!!! Question "How to edit the bot configuration?"
     You can edit the configuration at any time, which is available as `user_data/config.json` (within the directory `ft_userdata`) when using the above configuration.
```

Comment:
Maybe we should have another Question / option / whatever here pointing to the "edit compose-file" section ... 
The default is designed to work out of the box (defines SampleStrategy as --strategy) - but that'll not do it 99.99% of the users.

See also the note on the mkdocs.yml about combining this file with the compose  advanced section

Reply:
Added in the same question.